tests/gtests/test_iface_attr_quantization.cpp

```diff
@@ -84,7 +84,7 @@ TEST_F(attr_quantization_test_t, TestBNorm) {
                 eng, prop_kind::forward_inference, md, md, 0.1f, flags));
         CHECK_UNIMPL(batch_normalization_forward::primitive_desc(eng,
                 prop_kind::forward_inference, md, md, 0.1f, flags,
-                gen_attr_with_scales()));
+                gen_attr_with_scales(/* with_wei = */ false)));
```

Comment:
Does it mean that scales.has_default_values is not properly called in implementations?